src/lib/swapper/swappers/ZrxSwapper/utils/constants.ts

```diff
@@ -7,3 +7,4 @@ export const DEFAULT_SOURCE: SwapSource[] = [{ name: SwapperName.Zrx, proportion
 export const AFFILIATE_ADDRESS = '0xc770eefad204b5180df6a14ee197d99d808ee52d'
 export const OPTIMISM_L1_SWAP_GAS_LIMIT = '50000'
 export const OPTIMISM_L1_APPROVE_GAS_LIMIT = '10000'
+export const FEE_RECIPIENT = '0x90A48D5CF7343B08dA12E067680B4C6dbfE551Be' // DAO treasury
```

Comment:
im seeing the dao treasury address in a few places now (also 1inch swapper), should we find somewhere to put it? we have a good spot for fox and eth addresses in caip but it probably doesnt belong there

Reply:
good thinkin @woodenfurniture. let's go ahead and set this up for each of the DAO's safes on each chain.

https://forum.shapeshift.com/thread/dao-treasuries-and-multisigs-43646

Reply:
Done! https://github.com/shapeshift/web/pull/4392/commits/9e19fe7eda4135d7566a80e77cde736bfcc81941